tests/e2e/data/help-contents.ts

```diff
@@ -216,6 +216,10 @@ Commands:
                                                                  [aliases: stop]
   taq list accounts <sandboxName>           List the balances of all sandbox acc
                                             ounts
+  taq typecheck <sandboxName> [sourceFiles  Typecheck Michelson contracts
+  ...]                                                             [aliases: tc]
+  taq simulate <sandboxName> <sourceFile>   Run Michelson contracts as a simulat
+                                            ion                   [aliases: sim]
```

Comment:
There should also be test for the individual commands for the help menu. I think there should be two more entries here similar to helpContentsFlextesaPluginTypeCheck and helpContentsFlextesaPluginSimulate

Reply:
We will be moving all to a new plugin after discussion with @mweichert and @hu3man